csvqb/csvqb/writers/qbwriter.py

```diff
@@ -421,3 +422,25 @@ def _get_measure_uri(measure: QbMeasure) -> str:
         return _doc_rel_uri(f"measure/{measure.uri_safe_identifier}")
     else:
         raise Exception(f"Unmatched unit type {type(unit)}")
+
+
+def _get_about_url(cube: Cube) -> str:
+    # Todo: Dimensions are currently appended in the order in which the appear in the cube.
+    #       We may want to alter this in the future so that the ordering is from
+    #       least entropic dimension -> most entropic.
+    #       E.g. http://base-uri/observations/male/1996/all-males-1996
+    multi_measure_cols = []
+    aboutUrl = "#obs"
+    for c in cube.columns:
+        if isinstance(c, QbColumn):
+            c: QbColumn
+            if isinstance(c.component, QbDimension):
+                aboutUrl = aboutUrl + f"/{{+{c.uri_safe_identifier}}}"
+            elif isinstance(c.component, QbMultiMeasureDimension):
+                multi_measure_cols.append(c.uri_safe_identifier)
+            else: 
+                pass
+    if len(multi_measure_cols) != 0:
+        for m in multi_measure_cols:
```

Comment:
There should only ever by *one* multi measure column. So this shouldn't be a list.